src/store/view-store.ts

```diff
@@ -601,6 +601,11 @@ export class ViewStore {
 		return this.project.getElements().find(element => element.getHighlighted());
 	}
 
+	@Mobx.action
+	public getFirstPage(): Model.Page {
```

Comment:
Let's remove `getFirstPage` and `getLastPage`, related to the comments on https://github.com/meetalva/alva/pull/558/files#diff-d3a5c49b9ddc2f6f39d233553a4f9c7dR398, https://github.com/meetalva/alva/pull/558/files#diff-d3a5c49b9ddc2f6f39d233553a4f9c7dR390

Let's introduce `getNextPage` and `getPreviousPage` instead. Using those we could do:

```ts
const nextPage = store.getNextPage();

if (nextPage) {
  store.setActivepage(nextPage);
}
```